script/os-autoinst-setup-multi-machine

```diff
@@ -16,7 +16,9 @@ ensure_ip_forwarding() {
 }
 
 install_packages() {
-    zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs
+    for i in $(seq 3); do
+        retry zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs && break
```

Comment:
Hmm, the description says that it's *not* using retry :)

Reply:
retry is already installed in the environment we are testing: https://openqa.opensuse.org/tests/4049058#step/prepare/5 . And retry is part of an official repository so it's much more likely to be successful compared to a package from a devel repository which is also what failed in https://progress.opensuse.org/issues/158245 , see https://openqa.opensuse.org/tests/4049058#step/openqa_worker/7 , so I suggest to install retry if it isn't installed and then use it

```bash
    command -v retry >/dev/null || zypper -n in retry
    retry zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs
```

Reply:
Good point. It makes sense to assume retry is installed in the test environment then which it generally is so I'll simply use it.

Reply:
I think the suggestion was to install retry if it is not installed. I guess that test is not the only place where this script might be called.

Reply:
I didn't like that suggestion. But I also don't care enough to defend my position here so I changed it.